deploy/playbooks/examples/deploy_vagrant.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- hosts: all
+  user: vagrant
+  roles:
+    - celery
+    - common
+    - repos
+    - statsd
+  vars:
+     app_name: "chacra"
+     use_ssl: true
+     wsgi_file: wsgi.py
+     wsgi_callable: application
+     ansible_ssh_port: 22
+     restart_app: true
+     binary_root: "/opt/binaries"
+     repos_root: "/opt/repos"
+     branch: "master"
+     development_server: true
+     short_hostname: "3"
+     fqdn: "3.node.a"
+     api_user: "admin"
+     api_key: "secret"
+
+     # callbacks
+     # using an IP here because vagrant won't know about
+     # the hostnames of other vagrant machines
```

Comment:
we could fix this by re-doing the hosts files using the inventory as source of truth:

https://gist.github.com/rothgar/8793800

(for later maybe, this is not a hard req for this pull request)